playwright-e2e/tests/additional-applications.spec.ts

```diff
@@ -196,6 +195,46 @@ test.describe('Upload additional applications', () => {
       await expect(page.getByText('This is a confidential draft order and restricted viewing applies')).toBeVisible();
     });
 
+  test('CTSC uploads standard C2 application with no PBA', async ({ page,
+                                                          signInPage,
+                                                          additionalApplications,
+                                                          uploadAdditionalApplications,
+                                                          uploadAdditionalApplicationsApplicationFee,
+                                                          uploadAdditionalApplicationsSuppliedDocuments,
+                                                          submit    }) => {
+      caseName = 'CTSC standard C2 application ' + dateTime.slice(0, 10);
```

Comment:
Stroooooooong recommendation to use luxon (formerly moment.js) for all datetime nastiness. It solves everything.